Themes/default/languages/Admin.english.php

```diff
@@ -153,7 +153,7 @@
 $txt['ban_description'] = 'Here you can ban troublesome people either by IP, hostname, username, or email.';
 $txt['ban_add_new'] = 'Add new ban';
 $txt['ban_banned_entity'] = 'Banned entity';
-$txt['ban_on_ip'] = 'Ban on IP (e.g. 192.168.10-20.*)';
+$txt['ban_on_ip'] = 'Ban on IP (e.g. 19.2.168.10-20.*)';
```

Comment:
I think it would be better to give example for each (IPv4 : 192.168.x not like this, even i'm hardly understanding this.